project/Build.scala

```diff
@@ -239,7 +239,6 @@ object DottyBuild extends Build {
 
       libraryDependencies ++= Seq("com.storm-enroute" %% "scalameter" % "0.6" % Test,
         "com.novocode" % "junit-interface" % "0.11"),
-      testFrameworks += new TestFramework("org.scalameter.ScalaMeterFramework"),
```

Comment:
I removed this, because we never run `dotty-bench/test`, we use `dotty-bench/test:run`. The latter doesn't require the framework integration.